### ISSUE TITLE
Crete mailing address in Server Field tab and Hours of Operation

### ISSUE THREAD

**Issue by saoyag001** [apache/trafficcontrol#5290]
I'm submitting a ...
-  new feature / enhancement request

-  Traffic Ops
-  Traffic Portal

Requesting to add optional field Mailing address to include
Address
City
state
Zip
Contact

Also requesting hours of operation to make it easier to schedule vendors on site.

## Current behavior:
currently usign notes which doesnt work well for automation

## New behavior:
with alternate mailing addresses automation will be easier.

**Comment by mitchell852** [comment]
What are we talking about? Physical location? Server? somethng else?

**Comment by saoyag001** [comment]
PHysical location is where the server resides, however this may not be in line with the actual mailing address. So would prefer mailing address on the actual server tab.

**Comment by mitchell852** [comment]
Ideally imo we should probably create an address table and have FK references from the phys_location and servers tables to an address...

**Comment by ocket8888** [comment]
I don't think I'm understanding this right. If the mailing address of your physical location isn't the right mailing address, then why not just change it?

**Comment by mitchell852** [comment]
> I don't think I'm understanding this right. If the mailing address of your physical location isn't the right mailing address, then why not just change it?

i guess the mailing address for the server can be different than the mailing address for the server's physical location. is that right @saoyag001

**Comment by saoyag001** [comment]
yes. mailing addresses are usually main locations, while serves themselves may reside up to 50 miles away.

**Comment by ocket8888** [comment]
So there are multiple server locations that are different but can be reached by post at a single, other address? Would contact information on a Region work for that?

**Comment by mitchell852** [comment]
> So there are multiple server locations that are different but can be reached by post at a single, other address? Would contact information on a Region work for that?

Isn't that not granular enough @ocket8888 ?

- Division has many regions
- Region has many phys locations
- phys location has many servers

what i'm hearing is that a server can have an address different that the phys location that it belongs too.

**Comment by saoyag001** [comment]
region is greater than physical location. region can have 30 physical locations and 10 of them can have one mailing address. An address thats not associated with any physical locations.
